src/index.ts

```diff
@@ -57,8 +60,15 @@ async function gameIterator(path) {
       metric.processGame(Array.from(historyGenerator), metadata);
     }
   }
+
+  for (const metric of metrics) {
+    if (metric === metadataMetric) {
+      metric.aggregate();
+      metric.logResults();
+    }
+  }
```

Comment:
this doesn't make sense Just directly call .aggregate on metadataMetrci

Reply:
sure but p sure u wrote this code hahahaha

Reply:
yeah bc i was correcting your code